MycroftAI/mycroft-core
# Refactor - common bus connection method

## Description
Following on from #2758 this switches the remaining Services to use the new common method for connecting to the message bus. This should only simplify the current code bringing together 4 different ways of doing the same thing.

## How to test
Run Mycroft and ensure all Services start and connect to the message bus as expected.

Given the significance of changes to the startup flow of Services, each should be checked carefully.

## Contributor license agreement signed?
